testutil/apiload/main.go

```diff
@@ -0,0 +1,257 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+)
+
+const (
+	OutputFileTemplate = "result_%s_%s.json"
+)
+
+// Attack specifies a duration and a series of attackers, each with their own
+// frequency and targets.
+type Attack struct {
+	Duration  int         `json:"duration"`
+	Attackers []*Attacker `json:"attackers"`
+}
+
+// Frequency is 1/second and endpoints are relative to root
+type Attacker struct {
+	Name      string   `json:"name"`
+	Frequency int      `json:"frequency"`
+	Delay     float32  `json:"delay"`
+	Endpoints []string `json:"endpoints"`
+}
+
+var Config *config
+
+func main() {
+	var err error
+
+	Config, err = loadConfig()
+	if err != nil {
+		log.Fatalf("%v", err)
+	}
+
+	// Profiles load from a JSON file. The default profiles are in profiles.json.
+	profiles, err := loadProfiles()
+	if err != nil {
+		log.Fatalf("%v", err)
+	}
+
+	if Config.ListAttacks {
+		listAttacks(profiles)
+		return
+	}
+
+	if Config.Attack == "" {
+		fmt.Println("No attack specified. Specify an attack with -a.\n")
```

Comment:
Linters compain about the "extra" newline since Println already inserts one.  Perhaps use fmt.Print or fmt.Printf and have two `\n`.